go-controller/pkg/ovn/endpoints.go

```diff
@@ -43,7 +43,7 @@ func (ovn *Controller) getLbEndpoints(ep *kapi.Endpoints) map[kapi.Protocol]map[
 
 // AddEndpoints adds endpoints and creates corresponding resources in OVN
 func (ovn *Controller) AddEndpoints(ep *kapi.Endpoints) error {
-	klog.V(5).Infof("Adding endpoints: %s for namespace: %s", ep.Name, ep.Namespace)
```

Comment:
I think that the logging should be changed separately and make it consistent in all components, actually, it will be better if we define some logging policies and enforce them in all the code.
Maybe define a default loglevel (i.e. `klog.V(2).Infof`) and set it by default , so users can decide if they want to swith it off later?

Reply:
Right now we only use v(5) for debug, and default Infof logging. These logging messages should not have been at debug. We log this level of info at default log level for other things like pods. If we want to use other log levels or standardize things, we can do it in another PR.

Reply:
Fair enough